packages/sdk/test/unit/browser/teamsUserCredential.browser.spec.ts

```diff
@@ -61,6 +62,21 @@ describe("TeamsUserCredential Tests - Browser", () => {
 
   const fakeAccessToken = "fake-access-token";
 
+  const fakeAuthCodeTokenResponse: AuthenticationResult = {
+    authority: "fake-authority",
+    uniqueId: "fake-uniqure-id",
+    tenantId: "fake-tenant-id",
+    scopes: ["user.read"],
+    account: null,
+    idToken: "fake-id-token",
+    idTokenClaims: {},
+    accessToken: fakeSSOTokenFull,
```

Comment:
the token get from the auth code flow is not sso token, can we change to another name to avoid confusing?

Reply:
Fixed. Thx!